llama_stack/distribution/utils/exec.py

```diff
@@ -91,9 +43,10 @@ def get_conda_prefix(env_name):
 
 def in_notebook():
     try:
-        from IPython import get_ipython
+        from IPython.core.getipython import get_ipython
```

Comment:
these changes are gratuitous (not part of the PR), please undo everything that is not relevant to removing conda